app/assets/javascripts/app/views.js

```diff
@@ -55,8 +55,7 @@ app.views.Base = Backbone.View.extend({
       .html(this.template(presenter))
       .attr("data-template", _.last(this.templateName.split("/")));
 
-    // add avatar fallback if it can't be loaded
-    this.$el.find(this.avatars.selector).on("error", this.avatars.fallback);
+    this.setupAvatarFallback();
```

Comment:
It was possible with some small refactoring which even improved the code. The conversations view now adds the fallback to the added recipient instead of adding it every time to all recipients.